src/main/java/net/dv8tion/jda/api/interactions/components/selections/SelectOption.java

```diff
@@ -260,7 +261,7 @@ public boolean isDefault()
      * @return The attached emoji
      */
     @Nullable
-    public Emoji getEmoji()
+    public EmojiUnion getEmoji()
     {
         return emoji;
```

Comment:
This would be the only place that needs a cast. All the others could be removed.